src/satisfiers/index.ts

```diff
@@ -1,2 +1,3 @@
 export { default as originSatisfier } from './origin'
 export { challengeSatisfier } from './challenge'
+export { default as routeSatisfier } from './route'
```

Comment:
I would just export the const `ROUTE_CAVEAT_CONFIGS` as is without default from its file and then this can be consistent with the others

```suggestion
export { routeSatisfier } from './route'
```